ui/components/remove.js

```diff
@@ -0,0 +1,29 @@
+import DeleteModal from './delete-modal'
+
+export default function Remove({
```

Comment:
How repetitive would it be to leave this de-coupled? Another option would be to bake this into a Sidebar `<Footer />` component or similar which may be easier to test

Reply:
maybe the name of this component is misleading? maybe I should rename this to remove button? I think it is a great idea to break the sidebar footer into a `<Footer/>` component but this component is just the remove button - the positioning of the button is in the parent components